### PR TITLE
o2.at.sat: issues 100-103

### PR DESCRIPTION
**Pull request by aappling-usgs**

### CODE REVIEW

**Comment by lawinslow** [comment]
roxygen generate too?

**Comment by aappling-usgs** [comment]
yeah, i'm a dope. one sec.

File: man/metab.Rd [hunk 1]
```diff
@@ -52,7 +52,7 @@ Returns daily time series of gross primary production (GPP), respiration (R), an
 # fake data
 	datetime <- seq(as.POSIXct("2014-06-16 00:00:00", tz="GMT"),
      as.POSIXct("2014-06-17 23:55:00", tz="GMT"), length.out=288*2)
-	do.obs <- 2*sin(2*pi*(1/288)*(1:(288*2))+0.75*pi) + 8 + rnorm(288*2, 0, 0.5)
```
Comment (lawinslow) [inline man/metab.Rd]: @aappling-usgs not like the phase of the fake DO?

File: R/o2.at.sat.R [hunk 2]
```diff
@@ -60,46 +86,73 @@ o2.at.sat <- function(ts.data, baro, altitude=0, salinity=0, model='garcia'){
 	return(data.frame(datetime=ts.data$datetime, do.sat=dosat))  
 }
 
-#'@export
+
+#' @rdname o2.at.sat
+#' @export
 o2.at.sat.base <- function(temp, baro, altitude=0, salinity=rep(0,length(temp)), model='garcia'){
```
Comment (lawinslow) [inline R/o2.at.sat.R]: Based on our conversations, should the default be "garcia-benson"?

File: man/metab.Rd (hunk 1 above)
Comment (aappling-usgs) [inline man/metab.Rd]: this just happened when I built the doc - note that I didn't change metab.R, but that it didn't match metab.Rd until this PR

Comment (lawinslow) [inline man/metab.Rd]: Ha, ok, was just curious. Seemed like an un-related edit.

File: R/o2.at.sat.R (hunk 2 above)
Comment (aappling-usgs) [inline R/o2.at.sat.R]: I do think that garcia-benson is the preferred model. I kept the default at garcia to keep things from shifting (even more) under current user's feet, but if you're good with garcia-benson as the default, I'll change it.

File: man/metab.Rd (hunk 1 above)
Comment (aappling-usgs) [inline man/metab.Rd]: yeah, pretty random. i figured the change would have to happen sometime, so just committed it. i guess it was @rBatt who didn't like the phase of the fake data? =)

File: R/o2.at.sat.R (hunk 2 above)
Comment (lawinslow) [inline R/o2.at.sat.R]: Change it. Let's go with preferred.

Comment (lawinslow) [inline R/o2.at.sat.R]: Then I'll merge.

**Pull request merged**